src/langcheck/metrics/metric_inputs.py

```diff
@@ -456,6 +456,8 @@ def get_standard_metric_inputs_with_required_lists(
     | tuple[IndividualInputType, IndividualInputType] = None,
     reference_outputs: IndividualInputType
     | tuple[IndividualInputType, IndividualInputType] = None,
+    additional_inputs: dict[str, IndividualInputType] | None = None,
+    additional_input_name_to_prompt_var_mapping: dict[str, str] | None = None,
     required_params: list[str],
 ) -> tuple[MetricInputs, list[list[str]]]:
     """Create a metric inputs object with the standard parameters
```

Comment:
Same here, something like

Create a metric inputs object with the standard parameters
(i.e. generated_outputs, prompts, sources, reference_outputs) and the
specified additional parameters. This function also returns...